### PR TITLE
[jni] revise android build

### PR DESCRIPTION
**Pull request by lhs8928**
- remove jni/Android.mk file. Will use jni/Android.mk.in
 - Revise docs to use tools/package_android.sh when build for android

Signed-off-by: hyeonseok lee <hs89.lee@samsung.com>

### CODE REVIEW

**Review by DonghakPark — APPROVED** [review]
LGTM !!

**Comment by jijoongmoon** [comment]
I'm not sure if we do need this Android.mk. I think we need to change this to use the Android.mk.in. I mean using meson build.
If you check the meson_option.txt, the defulat of `nntr-num-threads` is 1 and for the android build, we are not set the tflite enable.

**Comment by lhs8928** [comment]
> I'm not sure if we do need this Android.mk. I think we need to change this to use the Android.mk.in. I mean using meson build. If you check the meson_option.txt, the defulat of `nntr-num-threads` is 1 and for the android build, we are not set the tflite enable.

Done :)

**Review by songgot — APPROVED** [review]
LGTM

**Review by myungjoo — APPROVED** [review]

File: docs/how-to-run-example-android.md [hunk 1]
```diff
@@ -23,44 +23,24 @@ $ export PATH=$PATH:${PWD}/android-ndk-r21d/
...
-Android.mk      iniparser  obj                   prepare_openblas.sh  tensorflow-1.13.1
-Application.mk  libs       prepare_iniparser.sh  prepare_tflite.sh
-$ ls libs/arm64-v8a
-libcapi-nntrainer.so  libccapi-nntrainer.so  libc++_shared.so  libnntrainer.so
-$ mv libs ../
+$ ./tools/package_android.sh
+$ ls builddir/android_build_result
+Android.mk  conf  examples  include  lib
+$ ls builddir/android_build_result/libs/arm64-v8a
+libcapi-nntrainer.so  libccapi-nntrainer.so  libc++_shared.so  libnnstreamer-native.so  libnntrainer.so
```
Comment (jijoongmoon) [inline docs/how-to-run-example-android.md:40]: May be add another way to build as usual as below.

```
meson builddir -Dbuildtype=android  ( other options )
cd builddir
ninja
ninja install
```

Comment (lhs8928) [reply docs/how-to-run-example-android.md:43]: Fixed

**Review by jijoongmoon — APPROVED** [review]
LGTM

**Pull request merged**